hackathon_site/dashboard/frontend/src/api/types.ts

```diff
@@ -42,3 +42,45 @@ export interface Category {
     max_per_team: number;
     unique_hardware_count: number;
 }
+
+/** Event API */
+export interface Profile {
+    id: number;
+    id_provided: boolean;
+    attended: boolean;
+    acknowledge_rules: boolean;
+    e_signature: string;
+    team: number;
+}
+
+interface UserWithoutProfile {
+    id: number;
+    first_name: string;
+    last_name: string;
+    email: string;
+}
+
+export interface User extends UserWithoutProfile {
```

Comment:
As I mentioned above, it might be a little cleaner to write this the other way around and do

```typescript
export interface User {
    id: number;
    first_name: string;
    last_name: string;
    email: string;
    profile: Profile;
}

type UserWithoutProfile = Omit<User, "profile">;
```